pom.xml

```diff
@@ -273,7 +227,21 @@
             <phase>validate</phase>
             <configuration>
               <rules>
-                <dependencyConvergence />
+                <dependencyConvergence>
+                  <!-- The items listed below have been verified non issue for allowing. -->
```

Comment:
These might be a non-issue right now... but any changes to dependencies in the future could make them become an issue again... so this should not be done over the exclusions, which were done on a case-by-case basis.

Reply:
There is no difference in handling of exclusions vs case by case ignoring in this section.  The key its its in a single place not scattered throughout the pom.  That makes it actually easier to support.

Reply:
see https://maven.apache.org/enforcer/enforcer-rules/dependencyConvergence.html.  In our case, we already override everything, so this suffices.

Reply:
These are not equivalent. The override controls the version pulled in by the resolver. The individual excludes ensure we have removed only those that are safely overridden by the version we choose. If any new ones show up in the dependency tree, we will need to verify that. If you suppress them in the rule, then we will not know that there are new occurrences to triage.